apitools/base/py/transfer.py

```diff
@@ -438,7 +440,7 @@ def GetRange(self, start, end=None, additional_headers=None,
             progress_end_normalized = True
         else:
             progress = start
-        while not progress_end_normalized or progress < end:
+        while not progress_end_normalized or progress < end + 1:
```

Comment:
two things: 

1. let's just do `progress <= end`.
2. this seems like it's an off-by-one error in all cases. @thobrla have we had any other reports of an extra byte being returned? or were you calling this in gsutil as though it didn't include `end`?